sass/_ResultActionsMenu.scss

```diff
@@ -0,0 +1,29 @@
+.CoveoResultActionsMenu {
+  display: none;
+  position: absolute;
+  right: 0;
+  border: 1px solid #ccc;
+  border-radius: 6px;
+  margin-top: -24px;
+  background: white;
+
+  &.coveo-menu-opened {
+    display: flex;
```

Comment:
Use mixin for everything that is related to flex.

@ include display(flex), @ include flex-flow(row nowrap), etc..